homeassistant/components/swiss_public_transport/coordinator.py

```diff
@@ -101,7 +104,7 @@ async def fetch_connections(self, limit: int) -> list[DataConnection]:
                 duration=calculate_duration_in_seconds(connections[i]["duration"]),
                 start=self._opendata.from_name,
                 destination=self._opendata.to_name,
-                remaining_time=str(self.remaining_time(connections[i]["departure"])),
+                remaining_time=self.remaining_time(connections[i]["departure"]),
```

Comment:
Just asking. Would it make more sense to return the total seconds of the time diff instead of a string that is always in English?
In Switzerland, most people are not native English speakers, and maybe they want to show the remaining time in their language, e.x. in German

Reply:
Yeah it might be good. Currently, the `remaining_time` field is not used by the `sensors` and only available through a service call. To display the `duration until next departure`, a timestamp sensor is used based on the `departure` field of the `DataConnection`.

Therefore, as it is only via service call, providing raw `int` value in seconds gives allows for best further transformation for the user.

Reply:
@polgarc I think we should switch it to ±`int` in seconds for the `remaining_time` and then we are good to go?

Reply:
Sure but then I'd argue that the original return type `timedelta` was better. If the hard coded English string is a concern, then I'd rather not change anything at all.

Reply:
It would leave it up to the user to interprete the timedelta and maybe it would be nice to have an indication if it is in the future or the past.